### DISCUSSION TITLE
rssi should be signal and vice versa

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#11130] [Q&A] [answered]
According to wikipedia (https://en.wikipedia.org/wiki/Received_signal_strength_indication) and IEEE 802.11 RSSI is the relative received signal strength in a wireless environment, in arbitrary units. RSSI is an indication of the power level being received by the receiving radio after the antenna and possible cable loss. Therefore, the greater the RSSI value, the stronger the signal. Thus, when an RSSI value is represented in a negative form (e.g. −100), the closer the value is to 0, the stronger the received signal has been.

Tasmota uses RSSI as a signal quality indicator instead and uses Signal as an RSSI indicator.

Those 2 values should switch places to be in line with the standards.

**Comment by ascillato** [comment, marked as answer]
Hi, thanks, this was already discussed in old issues and changing that will be a breaking change for a huge amount of users. Will be left as it is now. Sorry.

**Reply by sfromis** [reply]
The history is that Tasmota started out by converting RSSI to a "more user friendly" signal quality, up to 100%. The signal strength was added later, and then the "RSSI" label was taken.

**Comment by jeroenst** [comment]
Is it an idea to change this according the standards in the next major release?

**Reply by arendst** [reply]
Nope. See above.
